packages/awkward-cpp/meta.yaml

```
@@ -0,0 +1,27 @@
+package:
+  name: awkward-cpp
+  version: 22
+  top-level:
+    - awkward_cpp
+
+source:
+  url: https://pypi.io/packages/source/a/awkward-cpp/awkward-cpp-22.tar.gz
```

```suggestion
  url: https://files.pythonhosted.org/packages/8b/f8/97efcc2d52e2f9967f9beed86ed6736fc6eed91eb2d974a1bdc8c438146d/awkward-cpp-22.tar.gz
```

Let's use `https://files.pythonhosted.org` url to prevent redundant redirections.